Converted the Change Fastly credentials KB article to DevDocs (#5501)
magento/devdocs: guides/v2.2/cloud/cdn/cloud-fastly.md

On initial provisioning or upgrade of your {{ site. data.var.ece }} project, we install the latest version of the Fastly CDN module in your Staging and Production environments. When Fastly releases module updates, you receive notifications in the Magento Admin UI for your environments. We recommend that you update your environments to use the latest release. See [Upgrade Fastly]({{ page.baseurl}}/cloud/cdn/configure-fastly.html#upgrade).

## Fastly service account and credentials

{{ site.data.var.ece }} projects do not require a dedicated Fastly account or account owner. Instead, each Staging and Production environment has unique Fastly credentials (API token and service ID) to configure and manage Fastly services from the Magento Admin UI. You also need the credentials to submit Fastly API requests.

During project provisioning, Magento adds your project to the Fastly service account for {{ site.data.var.ece }} and adds the Fastly credentials to the configuration for the Staging and Production environments. See [Get Fastly credentials]({{ page.baseurl }}/cloud/cdn/configure-fastly.html#cloud-fastly-creds).

### Change your Fastly API token

> To change the Fastly API token credential for any reason, submit a Magento support ticket to request a new credential, and then update your Staging or Production environment with the new value.

Reviewer: It says to `change`, but then the section is about getting a new credential. I think it could be more clear, like to change the credential you need a new token?

```suggestion
Before you can change the Fastly API token credential for any reason, you must request a new Fastly API token. Submit a Magento support ticket to request a new token, and then update your Staging or Production environment with the new value.
```

> Before you can change the Fastly API token credential for any reason, you must request a new Fastly API token. Submit a Magento support ticket to request a new token, and then update your Staging or Production environment with the new value.

Author: ```suggestion
If you need to change the Fastly API token credential, you must submit a Magento support ticket to request a new token, and then update your Staging or Production environment with the new value.
```
